### ISSUE TITLE
Fill in approved recommendations

### ISSUE THREAD

**Issue by strogonoff** [ituob/itu-ob-data#26]
Turns out this is somewhat lower priority, as https://www.itu.int/dms_pub/itu-t/opb/sp/T-SP-OB.1171-2019-OAS-PDF-E.pdf shows even in the OB some recommendations do not have titles.

**Comment by ronaldtse** [comment]
@strogonoff I think we just need to accept that translated titles come in phases, asynchronously with the OBs. So it was the correct choice to keep data normalized and separate.

**Comment by strogonoff** [comment]
Here in the description my point was that some OBs don’t have titles inside. This shows that if some are missing at first that may not be a particularly huge concern, which is useful in prioritization.

**Comment by strogonoff** [comment]
and I mean English titles, if you looked in my example.

**Comment by ronaldtse** [comment]
@strogonoff did you mean this one doesn't have a title? "ITU-T G.709.1/Y.1331.1 (2018) Amd. 1 (04/2019)" 

It's because this document is an amendment to an existing one. The title of the original document "ITU-T G.709.1/Y.1331.1 (2018)" is unchanged.

**Comment by strogonoff** [comment]
@ronaldtse Well, even then it seems inconsistent that in some cases the titles are missing. See this page for example: https://www.itu.int/rec/T-REC-P.340/en. Some amendments have titles, some don’t.

This is looking to be a pain to deal with.

**Comment by ronaldtse** [comment]
@strogonoff wow. Anyway, we should refer to this page instead for this example: https://www.itu.int/ITU-T/recommendations/rec.aspx?rec=5078&lang=en . Here, both amendments have "titles", only the "Corrigendum" does not.

**Comment by strogonoff** [comment]
I see. I believe OB staff uses the resource I linked, and that’s why it doesn’t have title in the edition.

**Comment by ronaldtse** [comment]
> I see. I believe OB staff uses the resource I linked, and that’s why it doesn’t have title in the edition.

Maybe I'm misunderstanding, but the page you shared does show titles for Amendments:

<img width="774" alt="Screen Shot 2019-07-27 at 11 33 58 PM" src="https://user-images.githubusercontent.com/11865/62003130-03c91100-b0c7-11e9-9400-a2ab7e0d68c7.png">

**Comment by strogonoff** [comment]
No, it was me, wrong example. The amendment title is missing for another rec: https://www.itu.int/rec/T-REC-G.709.1/en. (Also missing in the OB for this rec.)

**Comment by ronaldtse** [comment]
Got it, good find. I suspect that only certain Amendments have titles. e.g. in the earlier one it was either adding a new Annex B or replacing the entire Annex B. Amendments that combine multiple changes (in G.709.1 Amendment 1 it contained a new Annex and many other changes), might not have an appropriate title.
